lib/translations.json

```diff
@@ -175,28 +187,40 @@
       "value": "{name} für"
     },
     {
-      "key": "checklist/labels/journalist",
-      "value": "Text fertig"
+      "key": "checklist/labels/startCreation",
+      "value": "Start Kreation"
+    },
+    {
+      "key": "checklist/labels/startProduction",
+      "value": "Start Produktion"
+    },
+    {
+      "key": "checklist/labels/numbersOk",
+      "value": "Grafik / Zahlen OK"
```

Comment:
slash is ugly, maybe just ` und ` or `Data OK`

Reply:
ok, will change it to 'und'